docs/multi-dc.md

```diff
@@ -12,11 +12,15 @@ enterprise: 'no'
 
 # Installing the Initial Datacenter
 
-Install the cluster as described in the Install and Customize section. No additional configuration is necessary.
+Install the cluster as described in the Install and Customize section. If all virtual datacenters in the Cassandra cluster will reside in the same DC/OS cluster, no additional configuration is necessary.
 
-Tthe `service.data_center_url` will automatically be set to the Mesos DNS address of the cluster (i.e., `http://<service.name>.marathon.mesos:<api_port>/v1/datacenter`) if left blank. For example, if the default service name and api port are used, the URL is set to `http://cassandra.marathon.mesos:9000/v1/datacenter`.
+The `service.data_center_url` will automatically be set to the Mesos DNS address of the cluster (i.e., `http://<service.name>.marathon.mesos:<api_port>/v1/datacenter`) if left blank. For example, if the default service name and api port are used, the URL is set to `http://cassandra.marathon.mesos:9000/v1/datacenter`.
 
-**Important:** We do not currently recommend creating Cassandra clusters that span multiple DC/OS clusters.
+If your Cassandra cluster spans multiple DC/OS clusters, perform the following steps to allow the Cassandra clusters to communicate with one another.
+
+1. Set up a reverse proxy in your first datacenter, accessible from other datacenters, that routes to the Mesos DNS address of the DC/OS cluster.
```

Comment:
Now that I think about it, each datacenter needs to do this, since messages will be sent both ways.